### ISSUE TITLE
Android Deferred deeplink testing for Reinstalls

### ISSUE THREAD

**Issue by shriharsha-bhagwat** [adjust/android_sdk#569]
Hello I have gone through many tickets related to testing of deferred deeplinks. But I still have a question specific to the case of Reinstallation.

How our deferred deeplink looks : https://app.adjust.com/{tracker_token}?deep_link=com.example.example://example.com/s/Bayern--Deutschland?adj_t=tracker_token&adj_campaign=test&adj_adgroup=test

What worked :
✅  First time install
✅  Reinstall but I forget my device in testing console and test this deferred deeplink, It works well. I am redirected to the specific page in our app.

What did not work :
⛔  Reinstall but I did not forget my device in testing console. Shouldn't this scenario also work while testing to make sure adjust deferred deeplink works well even for reinstalls ? How do we test on production for a real user scenario who would have reinstalled our app via deferred deeplink.

In short, how can I make sure to test deferred deeplink,  to be 100% sure that deferred deeplink works in all cases.

**Comment by shashanksu** [comment]
Hi @shriharsha-bhagwat ,
Sorry for the delay in responding.  Do you still have issues with deferred deeplink testing?  I would suggest to get in touch with your account support team from Adjust for this. 
Thanks

**Comment by shashanksu** [comment]
Hi @shriharsha-bhagwat,
Closing this as there are no further update.  But if you still having this issue, please get back.
Thanks
